test262/per-file/main.js

```diff
@@ -0,0 +1,225 @@
+"use strict";
+
+// (() => {
+let resultsNode;
+let summaryLabel;
+let summaryStatusLabel;
+let leafTreeNodeTemplate;
+let nonLeafTreeNodeTemplate;
+let pathInTree = [];
+let tree;
+const resultsObject = Object.create(null);
+
+function initialize(data, modeName) {
+  let mode;
+  if (modeName === "") {
+    mode = "AST";
+  } else if (modeName === "-bytecode") {
+    mode = "bytecode";
+  } else {
+    throw new Error("Unknown mode: " + modeName);
```

Comment:
can I interest you in some template literals

Reply:
you mean like this?
```js
mode = `${modeName == "" ? "AST" : modeName == "-bytecode" ? "bytecode" : (()=>throw new Error(`Unknown mode: ${modeName}`))()}`;
```

Reply:
uh, more like this:

```suggestion
    throw new Error(`Unknown mode: ${modeName}`);
```